packages/ckeditor5-list/src/list/utils.ts

```diff
@@ -7,16 +7,33 @@
  * @module list/list/utils
  */
 
-import { TreeWalker, getFillerOffset } from 'ckeditor5/src/engine';
-import { ButtonView } from 'ckeditor5/src/ui';
+import type { Editor } from 'ckeditor5/src/core';
+
+import {
+	TreeWalker,
+	getFillerOffset,
+	type DowncastConversionApi,
+	type DowncastWriter,
+	type Element,
+	type Item,
+	type Model,
+	type Position,
+	type ViewContainerElement,
+	type ViewElement,
+	type ViewItem,
+	type ViewPosition
+} from 'ckeditor5/src/engine';
+
+import { ButtonView, type ButtonExecuteEvent } from 'ckeditor5/src/ui';
+
+import '../listcommands';
```

Comment:
Why this is imported only here?

Reply:
Because this file relies on type inference in `commands.get()` and `CommandsMap` is augmented in `"../listcommands"`.